tests/unit/NameRequestInvalidErrorDialog.spec.ts

```diff
@@ -0,0 +1,95 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { shallowMount } from '@vue/test-utils'
+import { NameRequestInvalidErrorDialog } from '@/components/dialogs'
+import { NameRequestStates } from '@/enums/nameRequestStates'
```

Comment:
not 100% sure but if this was exported using the index file in the enums, it should be able to just import from `@/enums`

Reply:
Done!